zerver/lib/actions.py

```diff
@@ -4757,8 +4758,8 @@ def validate_email(user_profile: UserProfile, email: str) -> Tuple[Optional[str]
 
     try:
         validate_email_for_realm(user_profile.realm, email)
-    except ValidationError:
-        return None, _("Already has an account.")
+    except ValidationError as error:
+        return None, (error.code)
```

Comment:
let's call this `explanation`, not `code`; `code` in exceptions means something specific in Zulip.

Reply:
@timabbott  There is no keyword called explanation in a validation error. I think we can only use 
code, message, and attribute.

Reply:
Can I use attribute?

Reply:
@timabbott I think we can only use code as a parameter in django's ValidationErrors.
https://docs.djangoproject.com/en/2.1/ref/forms/validation/#raising-validationerror